docs/changelogs/CHANGELOG-2.21.md

```diff
@@ -0,0 +1,467 @@
+# Kubermatic 2.21
+
+- [v2.21.0](#v2210)
+
+## [v2.21.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.21.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.20/tutorials_howtos/upgrading/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure usercluster reconciliations will not cause resource exhaustion on seed clusters.
```

Comment:
This link should probably be https://docs.kubermatic.com/kubermatic/v2.21/tutorials-howtos/upgrading/, even if that is not available yet, it will be with https://github.com/kubermatic/docs/pull/1191.